docs/ProjectCommands.md

```diff
@@ -0,0 +1,64 @@
+# Project Command Reference
+
+Commands to do with projects. Remember that parameters with whitespace
+must be enclosed by quotation marks.
+
+## Options
+```sh
+/rocket project {list, view, help, create, unassign, edit, assign, delete}
+```
+
+### List
+```sh
+/rocket project list
```

Comment:
i like this

Reply:
oops, did I put an extra format line in there?